lib/config/screen.ex

```diff
@@ -23,7 +23,7 @@ defmodule ScreensConfig.Screen do
           | :pre_fare_v2
 
   @type t :: %__MODULE__{
-          vendor: :gds | :mercury | :solari | :c3ms | :outfront | :lg_mri | :mimo,
+          vendor: :gds | :mercury | :solari | :c3ms | :outfront | :lg_mri | :mimo | :n_a,
```

Comment:
Would it make sense to instead allow this to be `nil`/`null`, and have the existing "N/A" values serialize to that? It's ultimately still an atom that means "none", just a bit more blessed by the language. (It feels like initially we wanted to always require _some_ vendor, but with third-party screens, maybe that doesn't make sense anymore.)

Reply:
Yeah that makes sense. Totally blanked on that being an option.